docs/_data/portland-2022-config.yaml

```diff
@@ -210,15 +210,15 @@ sponsors:
 
 
 # Things that change over time, listed in order of change
-flaglanding: True
+flaglanding: False
 flaghassponsors: True
 flagcfp: True
-flagticketsonsale: False
+flagticketsonsale: True
 flagsoldout: False
 flagspeakersannounced: False
 flagrunofshow: False
 flaghasschedule: False
-flaghasunconf: False
+flaghasunconf: True
```

Comment:
@plaindocs is this for just the sign-up sheet? I may have incorrectly flipped this...